### PR TITLE
ch01-02-hello-world.md translated to spanish

### PR DESCRIPTION
**Pull request by Osovich**
Finished translation, translated around 1150 words this time to spanish. Let me know if there's something I should change!

### CODE REVIEW

**Comment by thecodix** [comment]
Hey @Osovich this PR has conflicts, do you mind solving them? Otherwise is not possible to merge

Come back to me if you have any more questions

**Comment by Osovich** [comment]
On it! Thanks for letting me know

**Comment by Osovich** [comment]
wait, how did that file get in there? there's something going on, the branches are acting strange, i'll try to find the problem for later translations! I fixed the file but it wasn't supposed to be there.

File: second-edition/src/appendix-05-translation.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
-## Apéndice E: Traducciones del Libro
+## Appendix E: Translations of the Book
```
Comment (thecodix) [inline second-edition/src/appendix-05-translation.md]: Coming back to english here! Please check 💃

Comment (Osovich) [reply second-edition/src/appendix-05-translation.md]: hahahahaha whaaat, let me fix, my way of using branches must be wrong, that's probably what's happening

**Comment by Osovich** [comment]
I think it's fixed, I'll try to solve this messy branching of mine before any other translation, sorry!

**Comment by thecodix** [comment]
No problems, was probably my fault for not merging before with the original docs

Hope you don't run into more issues 🙉

**Pull request merged**